src/project-css/app/Jobs/UpdateSearchIndex.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * @author Tracy A McCormick <tam0013@mail.wvu.edu>
+ */
+
+namespace App\Jobs;
+
+use App\Adapters\UpdateSearchAdapter;
+use Illuminate\Bus\Queueable;
+use Illuminate\Http\Request;
+use Illuminate\Queue\SerializesModels;
+use Illuminate\Queue\InteractsWithQueue;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Support\Facades\DB;
+
+use Log;
+
+class UpdateSearchIndex implements ShouldQueue
+{
+    use InteractsWithQueue, Queueable, SerializesModels;
+
+    private $tblNme;
+    private $records;    
+
+    /**
+     * Create a new job instance.
+     */
+    public function __construct($tblNme, $records)
+    {
+        $this->tblNme = $tblNme;
+        $this->records = $records;
+    } 
+
+    /**
+     * Execute the job.
```

Comment:
Needs Proper Documentation: 
Use full PHP Documentation Standards.
Be clear on what the parameters of the function are as well as what the return values are supposed to be.